packages/next/test/unit/utils.test.js

```diff
@@ -8,8 +8,8 @@ const {
 const { FileRef } = require('@vercel/build-utils');
 
 describe('getNextConfig', () => {
-  const workPath = path.join(__dirname, 'fixtures');
-  const entryPath = path.join(__dirname, 'fixtures', 'entry');
+  const workPath = path.join(__dirname, 'fixtures', '00-config');
```

Comment:
Was this intended?

Reply:
Yes. The fixtures directory seemed to contain a single fixture for testing the config stuff and since I needed to add a fixture for this PR, I moved the config stuff into a subdir and had to update the next config test. Now we can use the next/test/unit/fixtures dir for other unit test fixtures.

Reply:
I see now, I was confused because this was `utils.test.js` but it was really testing one of the fixtures.

We can probably do some more clean up here but that can wait for a future PR 🙂